src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -302,36 +293,42 @@ interface IOptionSettlementEngine {
     function option(uint256 tokenId) external view returns (Option memory optionInfo);
 
     /**
-     * @notice Returns Claim struct details about a given tokenId if that token is a
+     * @notice Returns OptionLotClaim struct details about a given tokenId if that token is a
      * claim NFT.
      * @param tokenId The id of the claim.
      * @return claimInfo The Claim struct for tokenId.
      */
-    function claim(uint256 tokenId) external view returns (Claim memory claimInfo);
+    function claim(uint256 tokenId) external view returns (OptionLotClaim memory claimInfo);
 
     /**
-     * @notice Returns the total amount of options written and exercised for all claims /
-     * option lots created on the supplied index.
+     * @notice Returns the total amount of options written and exercised for all
+     * option lot claims created on the supplied index.
      * @param optionId The id of the option for the claim buckets.
      * @param dayBucket The index of the claimBucket to return.
      */
     function claimBucket(uint256 optionId, uint16 dayBucket)
         external
         view
-        returns (ClaimBucket memory claimBucketInfo);
+        returns (OptionLotClaimBucket memory claimBucketInfo);
 
     /**
-     * @notice Updates the address fees can be swept to.
-     * @param newFeeTo The new address to which fees will be swept.
+     * @notice Information about the position underlying a token, useful for determining
```

Comment:
Can we further elucidate? maybe "when supplied and options lot claim id, underlying() will return the total amounts of the underlying and exercise assets currently associated with an options lot".

Reply:
I like it, updated.